src/core/common/Topbar/fragments/PageTitle.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { get } from "lodash";
+
+import Typography from '@material-ui/core/Typography';
+
+import PatientBanner from "./PatientBanner";
+
+const pagesWithTitle = [
+    '/',
+    '/charts',
+    '/patients',
+];
+
+const titlesArray = {
+    charts: "System Dashboard",
+    patients: "Patients Lists",
+};
+
+const PageTitle = ({ classes, location, patientInfo }) => {
+    const pathName = get(location, 'pathname', null);
+    if (-1 !== pagesWithTitle.indexOf(pathName)) {
```

Comment:
Yoda conditions are not idiomatic in JS, and ESLint warns the developer when they  use single equal in a predicate

Reply:
Yoda conditions were removed everywhere.